datafusion/sqllogictest/test_files/string/string_view.slt

```diff
@@ -91,6 +91,16 @@ select octet_length(column1_utf8view) from test;
 7
 NULL
 
+query IIII
+SELECT
+  BIT_LENGTH(arrow_cast('Andrew', 'Utf8View')),
+  BIT_LENGTH(arrow_cast('datafusion数据融合', 'Utf8View')),
+  BIT_LENGTH(arrow_cast('💖', 'Utf8View')),
+  BIT_LENGTH(arrow_cast('josé', 'Utf8View'))
+;
+----
+48 176 32 40
```

Comment:
I think we can move this test to `string/string_literal.slt` and then add similar tests for other string types as well (UTF8, LargeUTF8, and DictionaryString).